src/mail/model/MailModel.js

```diff
@@ -297,6 +303,18 @@ export class MailModel {
 					                               .then((newId) => this._showNotification(newId || mailId)))
 					           .catch(noOp)
 				}
+			} else if (isUpdateForTypeRef(MailTypeRef, update) && update.operation === OperationType.DELETE) {
```

Comment:
We have to move it to our new model

Reply:
Do you mean that we have to listen for entityEventUpdates in the new Model, or that we just use the Editors from the new Model?